docs/tools/vdb_table/data/pinecone.json

```diff
@@ -133,20 +133,20 @@
     "comment": ""
   },
   "sharding": {
-    "support": "",
-    "source_url": "",
+    "support": "full",
```

Comment:
This column is to indicate that there is support for sharding the index across multiple machines (horizontal scaling). Is this what happens under the hood in Pinecone? I only see details about vertical scaling here: https://www.pinecone.io/learn/testing-p2-collections-scaling/

Reply:
yes https://docs.pinecone.io/docs/scaling-indexes#horizontal-scaling — I will replace with this link

Reply:
thanks I had missed that